src/config/dark-sites.config

```diff
@@ -1205,34 +1116,31 @@ tripleaughtdesign.com
 trojansource.codes
 trovo.live
 truckersmp.com
+truffle.vip
 tsssaver.1conan.com
 tube.kdy.ch
 tukui.org
-tusharsadhwani.dev
 tvnz.co.nz
 tvplus.com.tr
+twitch.tv
 twiukraine.com
 twnft.vercel.app
 tycrek.com
-tygo-van-den-hurk.com
 tylerpalko.github.io
 tynker.com/ide
 tyt.com
 ufplanets.com
-ukui.org
 ultrajs.dev
 undergroundcellar.com
-underlords.com
```

Comment:
Works for me and should stay here!
![obraz](https://github.com/user-attachments/assets/42665bef-d5da-46a4-a4bd-3fa7b14951e6)